bobtemplates/plone/content_type/tests/test_ct_+dexterity_type_name_normalized+.py.bob

```diff
@@ -123,3 +126,63 @@ class {{{ dexterity_type_name_klass }}}IntegrationTest(unittest.TestCase):
         )
         self.assertTrue(IDexterityItem.providedBy(obj))
 {{% endif %}}
+
+{{% if dexterity_type_global_allow | lower == 'true' %}}
```

Comment:
I think the problem is mrbob.hooks.to_boolean converts the option to boolean but while rendering in jinja template that option becomes `u'False'` instead of pythonic boolean `False`. So I think, even a raw comparison with string is enough but I found out using `default` filter is better.